_2017_pages/schedule.html

```diff
@@ -4,332 +4,596 @@
+  box-sizing: border-box;
+}
+
+.cf_sign_on_button {
+  width: auto !important;
+}
+
+.mktoFormRow {
+  width: 100% !important;
+}
+
+.mktoOffset {
+  display: none !important;
+}
+
+.mktoForm .mktoButtonWrap .mktoSimple .mktoButton, .mktoButton {
+  color: #FFFFFF !important;
+  border:1 !important;
+  background:#008CFF !important;
+  padding:16px !important;
+  font-size:16px !important;
+  box-shadow: none !important;
+  text-shadow: none !important;
+  white-space: nowrap !important;
+  font-weight: normal !important;
+  border-radius: 0 !important;
+}
+
+.mktoForm .mktoHasWidth .mktoLayoutLeft {
+  width: 100% !important;
+}
+
+.mktoForm .mktoFormRow .mktoField {
+border: 1px solid #000 !important;
+}</style>
+</head>
+<body class="page-template-default page page-id-9251 h2o no-touch ng-scope animate-in" ng-app="h2oApp">
+
+
+
+
+
+<section class="events-schedule explain-event" id="agenda" style="padding-bottom: 0px;">
+    <div class="container">
+
+
+<h5 style="
+    width: 200px;
+">October 31, 2020</h5>
+        <div class="col-outer">
+
+                            <div class="col-grid">
+                                <span class="time">9:00am</span>
+
+                                                                        <div class="column bg-black we-inner-col-1" style="">
+                                            <p>Welcome &amp; Orientation</p>
+																						<span>9:00AM - 9:30AM</span>&ensp;<span class="utc-tz">UTC+0</span>
+																				    <br>
+																				    <span class="time-americas">2:00AM - 2:30AM</span>&ensp;<span class="time-americas utc-tz">UTC-7</span>
+                                                                                    </div>
+                                                                </div>
+
+                            <div class="col-grid">
+                                <span class="time">9:30am</span>
+
+                                                                        <div class="column bg-yellow we-inner-col-1" style="">
+                                            <p>Keynote by Chris Fonnesbeck</p>
+<a href="https://pymc-devs.github.io/pymcon/speakers#chris-fonnesbeck-r" target="https://pymc-devs.github.io/pymcon/speakers#chris-fonnesbeck-r"></a>
+																						<span>9:30AM - 10:15AM</span>&ensp;<span class="utc-tz" style="position:relative; left: 1px;">UTC+0</span>
+																				    <br>
+																				    <span class="time-americas">2:30AM - 3:15AM</span>&ensp;&nbsp;&nbsp;<span class="time-americas utc-tz">UTC-7</span>
+																						<br>
+																						<span style="color:orange; position:relative; top: 5px;">Q&amp;A</span>
+																						&nbsp;&nbsp;&nbsp;&ensp;&ensp;
+																						<span style="color:orange; position:relative; top: 5px;">10:15AM</span>&nbsp;
+																						<span class="utc-tz" style="color:orange; position:relative; top: 5px;">UTC+0</span>
```

Comment:
Chris Q&A will take place in the America's time zone

Reply:
fixed, I also moved Aki's Q&A to Europe tz